src/README.md

```diff
@@ -3,3 +3,7 @@ This is the source folder. This is where all back-end code will go.
 
 # Update
 Since the app is all in java, we will have the entire program here.
+
+# Documentation (Part 1)
+# Class diagram, functional specification, entity-relationship diagram
```

Comment:
Don't do double lines like this with the "#". It'll look ugly when github converts this into a page. I would rather you make it look something like 
# Documentation
- Class Diagram, functional specification, entity-relationship diagram.
<<link>>
Change this before I merge